### PR TITLE
Add CPU requirement

### PR DESCRIPTION
**Pull request by dinithiW**
Add new WDL requirement 'cpu'

### CODE REVIEW

File: wdl2cwl/main.py [hunk 1]
```diff
@@ -344,6 +344,18 @@ def convert(workflow: str) -> str:
             )
         )
 
+    if "cpu" in ast.task_runtime:
+
+        cpu = ""
+        if '"' not in ast.task_runtime["cpu"]:
+            cpu = "$(inputs." + ast.task_runtime["cpu"] + ")"
```
Comment (mr-c) [inline wdl2cwl/main.py:351]: Okay, but what if `cpu` is just a plain number, no `"`?

**Review by mr-c — CHANGES_REQUESTED** [review]

File: wdl2cwl/main.py (hunk 1 above)
Comment (dinithiW) [reply wdl2cwl/main.py:351]: Right. I didn't think of that. I'll modify the code

**Comment by dinithiW** [comment]
Modified code to handle inputs given as the cpu requirement. Did not add the code to handle numbers in cpu requirement since there's no test case yet.

File: wdl2cwl/tests/wdl_files/hisat2.wdl [hunk 2]
```diff
@@ -25,7 +25,11 @@ task Hisat2 {
     input {
         File inputR1
         File? inputR2
+<<<<<<< HEAD
```
Comment (mr-c) [inline wdl2cwl/tests/wdl_files/hisat2.wdl:28]: You have some "merge conflict" markers left in the code 🙂 They should be removed

**Review by mr-c — CHANGES_REQUESTED** [review]

**Comment by mr-c** [comment]
> Did not add the code to handle numbers in cpu requirement since there's no test case yet. 

Please duplicate the existing test file and in the copy modify the CPU requirement to be a plain number

**Comment by dinithiW** [comment]
Add new test case to handle cpu requirement given as numbers. Also added the same functionality for time_minutes

File: wdl2cwl/main.py [hunk 3]
```diff
@@ -358,13 +358,35 @@ def convert(workflow: str) -> str:
...
+        time_minutes: Union[str, int] = ""
         if '"' not in ast.task_runtime["time_minutes"]:
-            time_minutes = "$(inputs." + ast.task_runtime["time_minutes"] + "* 60)"
+            if ast.task_runtime["time_minutes"].replace('"', "") in input_names:
+                time_minutes = (
+                    "$(inputs."
+                    + ast.task_runtime["time_minutes"]
+                    + "* 60)".replace('"', "")
+                )
+            if ast.task_runtime["time_minutes"].replace('"', "").isnumeric():
```
Comment (mr-c) [inline wdl2cwl/main.py:369]: Should be `elif`

File: wdl2cwl/main.py [hunk 4]
```diff
@@ -358,13 +358,35 @@ def convert(workflow: str) -> str:
...
+            )
+        )
+
+    if "cpu" in ast.task_runtime:
+
+        cpu: Union[str, int] = ""
+        if '"' not in ast.task_runtime["cpu"]:
+            if ast.task_runtime["cpu"].replace('"', "") in input_names:
+                cpu = "$(inputs." + ast.task_runtime["cpu"] + ")".replace('"', "")
+            if ast.task_runtime["cpu"].replace('"', "").isnumeric():
```
Comment (mr-c) [inline wdl2cwl/main.py:384]: should be `elif`

File: wdl2cwl/main.py [hunk 5]
```diff
@@ -358,13 +358,35 @@ def convert(workflow: str) -> str:
 
     if "time_minutes" in ast.task_runtime:
 
-        time_minutes = ""
+        time_minutes: Union[str, int] = ""
         if '"' not in ast.task_runtime["time_minutes"]:
-            time_minutes = "$(inputs." + ast.task_runtime["time_minutes"] + "* 60)"
+            if ast.task_runtime["time_minutes"].replace('"', "") in input_names:
```
Comment (mr-c) [inline wdl2cwl/main.py:363]: Is this `replace` necessary? We are already inside the `if '"' not in ast.task_runtime["time_minutes"]` block

File: wdl2cwl/main.py [hunk 6]
```diff
@@ -358,13 +358,35 @@ def convert(workflow: str) -> str:
...
-                timelimit=time_minutes.replace('"', ""),
+                timelimit=time_minutes,
+            )
+        )
+
+    if "cpu" in ast.task_runtime:
+
+        cpu: Union[str, int] = ""
+        if '"' not in ast.task_runtime["cpu"]:
+            if ast.task_runtime["cpu"].replace('"', "") in input_names:
```
Comment (mr-c) [inline wdl2cwl/main.py:382]: Is this `replace` necessary? We are already inside the ` if '"' not in ast.task_runtime["cpu"]` block

File: wdl2cwl/main.py [hunk 7]
```diff
@@ -358,13 +358,35 @@ def convert(workflow: str) -> str:
...
+                cpu = "$(inputs." + ast.task_runtime["cpu"] + ")".replace('"', "")
```
Comment (mr-c) [inline wdl2cwl/main.py:383]: Is this replace necessary?

**Review by mr-c — CHANGES_REQUESTED** [review]

**Review by mr-c — APPROVED** [review]

**Pull request merged**